Mathlib/Analysis/Distribution/TestFunction.lean

```diff
@@ -194,6 +205,126 @@ instance {R} [Semiring R] [Module R F] [SMulCommClass ℝ R F] [ContinuousConstS
     Module R 𝓓^{n}(Ω, F) := fast_instance%
   DFunLike.coe_injective.module R (coeFnAddMonoidHom Ω F n) fun _ _ ↦ rfl
 
+instance {R S} [Semiring R] [Semiring S] [Module R F] [Module S F] [SMulCommClass ℝ R F]
+    [SMulCommClass ℝ S F] [ContinuousConstSMul R F] [ContinuousConstSMul S F] [SMul R S]
+    [IsScalarTower R S F] :
+    IsScalarTower R S 𝓓^{n}(Ω, F) where
+  smul_assoc _ _ _ := by ext; simp
+
 end Module
 
+open ContDiffMapSupportedIn
+
+/-- The natural inclusion `𝓓^{n}_{K}(E, F) → 𝓓^{n}(Ω, F)` when `K ⊆ Ω`. -/
+@[simps -fullyApplied]
+def ofSupportedIn {K : Compacts E} (K_sub_Ω : (K : Set E) ⊆ Ω) (f : 𝓓^{n}_{K}(E, F)) :
```

Comment:
I have also used just `hK` as variable name for `K_sub_Q`: that's entirely subjective.